api/tests/core/offerers/test_api.py

```diff
@@ -386,6 +386,21 @@ def test_no_venue_contact_add_contact(self, app) -> None:
             "contact.website": {"new_info": contact_data.website, "old_info": "None"},
         }
 
+    def test_edit_reimbursement_point_id(self):
+        user = users_factories.UserFactory()
+        venue = offerers_factories.VenueFactory(pricing_point="self")
+        reimbursement_point = offerers_factories.VenueFactory(managingOfferer=venue.managingOfferer)
+        finance_factories.BankInformationFactory(venue=reimbursement_point)
+        venue_data = {
+            "reimbursementPointId": reimbursement_point.id,
+        }
+        offerers_api.update_venue(venue, author=user, **venue_data)
+
+        new_link = offerers_models.VenueReimbursementPointLink.query.one()
+        assert new_link.venue == venue
+        assert new_link.reimbursementPoint == reimbursement_point
+        assert new_link.timespan.upper is None
```

Comment:
(détails / question de goût) plus court et clair, et qui teste la même chose : 

```suggestion
        assert venue.current_reimbursement_point_id == reimbursement_point.id
```